src/middleware/checkAdmin.js

```diff
@@ -0,0 +1,14 @@
+const checkAdmin = (req, res, next) => {
```

Comment:
Since we already have an `adminPermission.js` in the middleware, it would be better if you put this function in that file. instead of having separate files for functions that do almost the same things

Reply:
@Malaba6, Done